frontend/app/models/transportation-analysis.js

```diff
@@ -95,9 +95,9 @@ export default class TransportationAnalysisModel extends Model {
   }
 
   // Fast Food boolean
-  @computed('project.commercialLandUse.[]')
-  get hasFastFood() {
-    return !!this.get('project.commercialLandUse').findBy('type', 'fast-food');
+  @computed('project.commercialLandUse.[]', 'fastFoodSqFt')
```

Comment:
nix `project.commercialLandUse.[]` from the CP signature, since it isn't used in the CP.